src/project/server/services/pokemon-helpers.js

```diff
@@ -0,0 +1,60 @@
+const {Todos}  = require('../db/models')
+const generateUniqueId = require('./generate-unique-id')
+const addTodoData = require('./add-todo-data.js')
+
+module.exports = async function handleAddSingleOrMultiPokemonsTodo(pokemonClient, enterValue){
+
+    if(enterValue.includes(",")){
+        const split = enterValue.split(",")
+        const pokemonArr = []
+
+        for(let i = 0; i < split.length; i++){
+            pokemonArr.push(pokemonClient.fetchMulti(sanitize(split[i])))
+        }
+        await handleAddMultiPokemonsTodo(pokemonClient, pokemonArr)
+    }
+    else {
+        await handleAddSinglePokemonTodo(pokemonClient, enterValue)
+    }
+}
+
+async function handleAddMultiPokemonsTodo(pokemonClient, pokemonArr){
+
+    return Promise.all(pokemonArr)
+        .then(async (response) => {
```

Comment:
why not use await? The mix of await with .then is weird - choose a convention and stick to it